azdo-task/vss-extension.json

```diff
@@ -2,7 +2,7 @@
     "manifestVersion": 1,
     "id": "ci",
     "name": "Dev Container Build and Run Task",
-    "version": "0.2.0",
+    "version": "0.0.0",
```

Comment:
Is this correct?

Reply:
`0.0.0` is the latest value on `main` (https://github.com/devcontainers/ci/blob/main/azdo-task/vss-extension.json#L5) from the PR to enable building on forks.

I'm confused about why this is showing as a change in your PR though

Reply:
There must have been a merge right after I synced. May just be bad timing?